### ISSUE TITLE
Weird results using pretrained model

### ISSUE THREAD

**Issue by zhuhaozh** [tomguluson92/PRNet_PyTorch#10]
Using the inference.py, got some weird results on my images. 

![image](https://user-images.githubusercontent.com/10545746/71705580-74651e00-2e1b-11ea-8272-ed19935d2fdc.png)
![image](https://user-images.githubusercontent.com/10545746/71705610-9e1e4500-2e1b-11ea-8781-960bfaf4d0be.png)

**Comment by eshuka** [comment]
I have a same problem. did you fix it?

**Comment by Torreto666** [comment]
I thinks the loss function has some problems

**Comment by Torreto666** [comment]
I think the loss function has some problems

**Comment by changfali** [comment]
Got that problem,too. Do you have some ideas? @Torreto666 @eshuka @tomguluson92

**Comment by punitha-valli** [comment]
@zhuhaozh @Torreto666 @changfali @eshuka @tomguluson92 

How do you get the depth map,
Can you please clear it 
Hope for a reply

Thanks in advance

**Comment by Jason0917** [comment]
Same problem here. Any thoughts on this issue?  XD

**Comment by tomguluson92** [comment]
Hi, everyone, the current model here is only trained on a few faces with extreme poses, so the model is not generalizing well to the common situations.

Train this PRNET with full dataset is all you need :)

**Comment by xuncheng1997** [comment]
> Hi, everyone, the current model here is only trained on a few faces with extreme poses, so the model is not generalizing well to the common situations.
> Train this PRNET with full dataset is all you need :)

When I train on full datasets, the results are still not good. How can i solve it.

**Comment by shiwen-cheng** [comment]
I have the same problem, has there anyone solved this?
please! i really need help :)
